[2021-12-17 15:33:49,685] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2021-12-17T14:33:24.198229+00:00 [queued]>
[2021-12-17 15:33:49,702] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2021-12-17T14:33:24.198229+00:00 [queued]>
[2021-12-17 15:33:49,702] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-17 15:33:49,702] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-17 15:33:49,702] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-17 15:33:49,719] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): accurate> on 2021-12-17 14:33:24.198229+00:00
[2021-12-17 15:33:49,720] {standard_task_runner.py:52} INFO - Started process 12907 to run task
[2021-12-17 15:33:49,723] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag', 'accurate', 'manual__2021-12-17T14:33:24.198229+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpizck9nsh', '--error-file', '/tmp/tmpi_wvfnxt']
[2021-12-17 15:33:49,723] {standard_task_runner.py:77} INFO - Job 18: Subtask accurate
[2021-12-17 15:33:49,756] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.accurate manual__2021-12-17T14:33:24.198229+00:00 [running]> on host sofienkhanfir-VirtualBox
[2021-12-17 15:33:49,789] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2021-12-17T14:33:24.198229+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-17T14:33:24.198229+00:00
[2021-12-17 15:33:49,789] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-17 15:33:49,790] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2021-12-17 15:33:49,795] {subprocess.py:85} INFO - Output:
[2021-12-17 15:33:49,796] {subprocess.py:89} INFO - accurate
[2021-12-17 15:33:49,796] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-17 15:33:49,814] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=accurate, execution_date=20211217T143324, start_date=20211217T143349, end_date=20211217T143349
[2021-12-17 15:33:49,868] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-17 15:33:49,895] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
